src/ape/types/abstract.py

```diff
@@ -50,7 +50,7 @@ def to_dict(v: Any) -> Optional[Union[list, dict, str, int, bool]]:
         return v
 
     else:
-        raise  # Unhandled type
+        raise Exception(f"Unhandled type '{type(v)}'")
```

Comment:
supercedes https://github.com/ApeWorX/ape/pull/80 and fixes https://github.com/ApeWorX/ape/issues/71